.idea/discord.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this file?

Reply:
its for phpstorm discord RPC can you remove it from the pr please?